contributingGuides/HOW_TO_BECOME_A_CONTRIBUTOR+_(C+).md

```diff
@@ -0,0 +1,24 @@
+## What is a Contributor+ (C+)?  
+C+ are contributors who have worked with Expensify for a while who review proposals in Github issues and pull requests (PRs) from contributors once they’ve been hired.  They recommend proposals and PRs then an internal Contributor Manager Engineer (CME) approves them.  
+
+## Why would someone want to be a C+
+- C+ are compensated the same price as the contributor for reviewing proposals and the associated PR. (ie. if a job is listed at $1000, that’s how much the C+ will make if they review both the proposals and PR).  If regressions are found that should have* been caught after the PR has been approved, C+ payment is reduced by 50% for each regression found.
+  - * Should have = C+ should have caught the bug by fully following the PR checklist.  If C+ skips a step or completed the checklist incompletely, payment will be cut in half.   
+- C+ can also work on jobs as a contributor
+- Earning potential is variable, it depends on how much a C+ wants to work and other jobs they’re hired for.  We’ve seen C+ make ~$100k/year.  
+- There isn’t a set number of hours a C+ needs to work in a week. Proposals and PRs reviews are expected to be addressed within 24 hours on weekdays. 
+- Dedicated #contributor-plus Slack room to discuss issues, processes and proposals. 
+- Future unknown benefits. 
```

Comment:
I'd just nix this:

```suggestion
```

Reply:
I'm torn cuz we're likely going to be inviting all C+ to Mexico soon.  And, if it's a success, we'd likely do it again

Reply:
Meh, my two cents is let's not leave vague promises here. Once we've done it, then we can include something like:

```
- In 2023, we invited all our Contributor+ to an exclusive, all-expenses-paid Expensify Open Source conference in Mexico. We're likely to continue to have other such special perks in the future.
```